lib/AST/Type.cpp

```diff
@@ -4085,3 +4085,67 @@ Type TypeBase::openAnyExistentialType(ArchetypeType *&opened) {
   opened = ArchetypeType::getOpened(this);
   return opened;
 }
+
+AnyFunctionType *AnyFunctionType::getAutoDiffAdjointFunctionType(
+    const AutoDiffParameterIndices &indices, const TupleType *primalResultTy) {
```

Comment:
Per convention, we generally do not add `const`s to type pointers in the swift code base.